plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/checkout-express-payment.tsx

```diff
@@ -76,30 +93,47 @@ const CheckoutExpressPayment = () => {
 
 	return (
 		<>
-			<LoadingMask
-				isLoading={
-					isCalculating ||
-					checkoutProcessing ||
-					isExpressPaymentMethodActive
-				}
+			<div
+				className={ clsx(
+					'wc-block-components-express-payment',
+					'wc-block-components-express-payment--checkout',
+					{
+						'wc-block-components-express-payment--disabled':
```

Comment:
From an accessibility point of view, this change is invisible to screen readers. I don't think this is a major problem, as it is not very likely that users would end up on this element while any of the pending operations are happening. Regardless of this aspect, it shows its downsides in testing, where you need to do some manual DOM querying to verify whether this class is there.

We could consider using some aria attributes. `aria-disabled` won't be correct as it should target only interactive elements, but `aria-hidden` is something that could work, both for the screen reader case and test improvements.

Reply:
Nice catch! I added some aria attributes and updated the test